common/workunit/workunit.cpp

```diff
@@ -5231,6 +5250,8 @@ void CLocalWorkUnit::copyWorkUnit(IConstWorkUnit *cached, bool all)
         CLocalWUResult result(LINK(&results->query()));
         result.setResultStatus(ResultStatusUndefined);
     }
+
+    copyTree(p, fromP, "usedsources"); // field usage info for enforcing column-level security
```

Comment:
Where in the workunit XML will this be?  I presume a branch off the root of the WU?